### ISSUE TITLE
 Remove `type: ignore` when inconsistencies between ophyd and bluesky are resolved

### ISSUE THREAD

**Issue by d-perl** [DiamondLightSource/dodal#827]
see #826 and depends on:
-  https://github.com/bluesky/bluesky/issues/1809

**Comment by stan-dot** [comment]
@callumforrester is this still an issue? both #826 and that other issue have been resolved

**Comment by callumforrester** [comment]
@stan-dot Yes, the job now is to remove all of the `type: ignore`s added in https://github.com/DiamondLightSource/dodal/pull/830

**Comment by stan-dot** [comment]
> git switch 827-remove-type-ignore-when-inconsistencies-between-ophyd-and-bluesky-are-resolved
fatal: invalid reference: 827-remove-type-ignore-when-inconsistencies-between-ophyd-and-bluesky-are-resolved

is the branch name too long?
